src/scenekit.cs

```diff
@@ -1550,19 +1722,20 @@ interface SCNScene : NSSecureCoding {
 		[Static, Wrap ("FromFile (name, directory, options == null ? null : options.Dictionary)")]
 		SCNScene FromFile (string name, string directory, SCNSceneLoadingOptions options);
 
-#if MONOMAC
-		[Mac (10,9), iOS (8,0)]
+		// Keeping here the same name WriteToUrl for iOS and friends because it is how it was bound
+		// initialy for macOS and having it named diferently would hurt shared code
+		[TV (10, 0), NoWatch, Mac (10, 9), iOS (10, 0)]
 		[Export ("writeToURL:options:delegate:progressHandler:")]
 		[EditorBrowsable (EditorBrowsableState.Advanced)]
 		bool WriteToUrl (NSUrl url,
 			[NullAllowed] NSDictionary options,
-			[NullAllowed] SCNSceneExportDelegate handler,
+			[NullAllowed] ISCNSceneExportDelegate handler,
```

Comment:
Yes, that would be a Mac-only fix (no need to have this error introduced on other platforms) and it's true that it's less an issue on macOS as it would be happen at JIT time (only if executed) and not at build time (e.g. AOT time or when the linker, optionally, process it).

Still ambiguities at compile time can be fixed by the developer (small source change). However *any* binary (which does not have to come from our store) using the symbol can fail, without an easy workaround.

As we are on a clock for iOS (not macOS) please open a bug report so it's not forgotten later.